wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -463,6 +471,10 @@ public void initialize() throws IOException, ModelException {
         downloadModel();
         loadServingProperties();
         downloadS3();
+        isTrtLlmBackend = "TRT-LLM".equals(Utils.getenv("LMI_BACKEND"));
+        if (isTrtLlmBackend) {
+            initTrtLlmModel();
```

Comment:
Please move these changes to the python engine. Probably the `PyModel.load()` function. The ModelInfo is a higher abstraction that shouldn't be modified by these changes. It should also be easier to test because you can just test it by loading and predicting with the Python Engine using the standard DJL predictor API

Reply:
Agree to your point. Will refactor this in a separate PR as I don't have the bandwidth to refactor and test it currently.